topics/microbiome/tutorials/dada-16S/tutorial.md

```diff
@@ -0,0 +1,1050 @@
+>        - In *"Filtering parameters"*:
+>            - *"Remove reads by number expected errors"*: `2`
+>
+{: .hands_on}
+
+> <comment-title>Speeding up downstream computation</comment-title>
+> 
+>  The standard filtering parameters are starting points, not set in stone. If you want to speed up downstream computation, consider tightening the *"Remove reads by number expected errors"* value. If too few reads are passing the filter, consider relaxing the value, perhaps especially on the reverse reads (e.g. *"Remove reads by number expected errors"* to `2` for forward and `5` for reverse), and reducing the *"Truncate read length"* to remove low quality tails. Remember though, when choosing *"Truncate read length"* for paired-end reads you must maintain overlap after truncation in order to merge them later. 
+> 
+{: .comment}
+
+> <comment-title>ITS data</comment-title>
+> 
+> For ITS sequencing, it is usually undesirable to truncate reads to a fixed length due to the large length variation at that locus. That is OK, just leave out  *"Truncate read length"*. See the [DADA2 ITS workflow](https://benjjneb.github.io/dada2/ITS_workflow.html) for more information 
+> 
+{: .comment}
+
+</div>
+
+**dada2: filterAndTrim** generates 2 outputs:
+- 1 paired-collection named `Paired reads` with the trimmed and filtered reads
+- 1 collection with statistics for each sample
+
+   > <question-title></question-title>
+   >
+   > 1. How many reads were in F3D0 sample before Filter and Trim?
+   > 2. How many readsa are in F3D0 sample before Filter and Trim?
+   >
+   > > <solution-title></solution-title>
+   > >
+   > > In F3D0 element in the collection, we see:
+   > > 1. `reads.in`: 7793
+   > > 2. `reads.out`: 7113
+   > >
+   > {: .solution}
+   >
+   {: .question}
+
+To better see the impact of filtering and trimming, we can inspect the read quality profiles and compare to the profiles before.
+
+<div class="Step-by-step" markdown="1">
+
+> <hands-on-title>Inspect read quality profiles after Filter And Trim</hands-on-title>
+>
+> 1. {% tool [dada2: plotQualityProfile](toolshed.g2.bx.psu.edu/repos/iuc/dada2_plotqualityprofile/dada2_plotQualityProfile/1.28+galaxy0) %} with the following parameters:
+>    - *"Processing mode"*: `Joint`
+>        - *"Paired reads"*: `paired - in a data set pair`
+>            - {% icon param-collection %} *"Paired short read data"*: `Paired reads` output of **dada2: filterAndTrim**
+>        - *"Aggregate data"*: `No`
+>    - *"sample number"*: `10000000`
+>
+{: .hands_on}
+
+</div>
+
+> <question-title></question-title>
+>
+> Looking at outputs of **plotQualityProfile** for reverse reads before and after **dada2: filterAndTrim**
+> 
+> {% snippet faqs/galaxy/features_scratchbook.md %}
+>
+> ![Quality Profiles for every samples with read length on the X-axis and Quality score on the Y-axis](./images/plotQualityProfile_reverse.png "Quality profiles for reverse reads before trimming and filtering")
```

Comment:
Same as above.

Reply:
every samples -> the forward/reverse reads of all samples

Reply:
Modified. Thanks